js/data/FinancialTimes_Bram-van-osta/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "financial_times_bram_van_osta",
+  "version": "0.0.1",
+  "description": "ECV exercice using Github API",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Bram van Osta",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-preset-es2015": "^6.18.0",
+    "babelify": "^7.3.0",
+    "browserify": "^13.1.1",
+    "gulp": "^3.9.1",
```

Comment:
Gulp shouldn't be necessary.  I'd recommand reading :
https://github.com/FrontenderMagazine/task_automation_with_npm_run/blob/master/eng.md
http://naholyr.fr/2015/11/ecrire-des-scripts-npm-multi-plateforme/
https://gist.github.com/oncletom/1e233f2100c4e0877922